github.go

```diff
@@ -16,6 +16,8 @@ var _ = github.ScopeAdminOrg
 var _ CommentService = &GitHubPullRequest{}
 var _ DiffService = &GitHubPullRequest{}
 
+var X = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var X should have comment or be unexported